storage/cleanup.go

```diff
@@ -194,6 +232,17 @@ func (m *cleanupManager) cleanupNamespaceFilesetFiles(earliestToRetain time.Time
 	return multiErr.FinalError()
 }
 
+func (m *cleanupManager) cleanupNamespaceSnapshotFiles(shards []databaseShard) error {
```

Comment:
same here: `cleanupShardSnapshotFiles`?

Reply:
same comment, willing to change it if you feel strongly